tests/Feature/GraphQL/Queries/GetCollectionsTest.php

```diff
@@ -45,6 +46,29 @@ public function test_it_can_fetch_with_no_args(): void
         $this->assertTrue(count($response['edges']) > 0);
     }
 
+    public function test_it_can_get_total_count_correctly(): void
+    {
+        $this->artisan('migrate:fresh');
+
+        $collections = Collection::factory(2)->create();
+        $collection1 = $collections->shift();
+        Token::factory(10)->create(['collection_id' => $collection1->id]);
+        CollectionAccount::factory(10)->create(['collection_id' => $collection1->id]);
+        $collection2 = $collections->shift();
+        Token::factory(10)->create(['collection_id' => $collection2->id]);
+        CollectionAccount::factory(10)->create(['collection_id' => $collection2->id]);
```

Comment:
Should we use bigger numbers in the tests?  The UI was showing no more than 15 even if there were many more items, so I'm guessing that any number up to 15 would likely pass the test regardless.

Reply:
Yes I can increase, i made it smaller purposely so it doesn't have big impact on the unit tests performance.